### DISCUSSION TITLE
RLC-811A FW v3.1.0.2109_23051509

### DISCUSSION THREAD

**Discussion by IAmVespa** [AT0myks/reolink-fw-archive#27] [Firmware sharing]
Given by Reolink support in order to fix IR lights not turning off during day time (did not work).
Said to be official latest firmware for RLC-811A, not yet on website because: "_The firmware sent is the latest version, sorry that our engineers have not released it to the Reolink website, you can install it on other RLC-811A cameras._"

**Changes** (compared to v3.1.0.989): 
- New web interface
- Possibility to connect IOT devices (eg. floodlight)
- Ability to turn on _"Illegal login lockout"_ under Advanced / User Management

Official Reolink [link](https://support-d.reolink.com/attachments/token/2b7ckPNIxn5zkQsdd3l5OkGcf/?name=IPC_523128M8MP.2109_23051509.RLC-811A.IMX415.8MP.AF.REOLINK.pak).

**Comment by AT0myks** [comment]
Two months later (sorry for the delay) the firmware is still not on their website. I'm afraid the engineers were lost trying to fix the IR lights.
Thanks for sharing and for the changelog. Added in 8d713d902e47c224e5161338aedd624b6a977ddf.
